modules/local/build_intervals.nf

```diff
@@ -0,0 +1,31 @@
+process BUILD_INTERVALS {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "anaconda::gawk=5.1.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gawk:5.1.0' :
+        'quay.io/biocontainers/gawk:5.1.0' }"
+
+    input:
+    tuple val(meta), path(index)
+
+    output:
+    tuple val(meta), path("*.bed")  , emit: bed
+    path "versions.yml"                     , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    prefix   = task.ext.prefix ?: "${meta.id}"
+
+    """
+    awk -v FS='\t' -v OFS='\t' '{ print \$1 }' ${index} > ${prefix}.bed
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        gawk: \$(awk -Wversion | sed '1!d; s/.*Awk //; s/,.*//')
```

Comment:
Is that correct? the `-Wversion`?

Reply:
Could just hardcde the version if it's wierd AF

Reply:
Seems to work fine